docs/developers/contributing.rst

```diff
@@ -0,0 +1,129 @@
+Contributing
+============
+
+When contributing any keylime repository, please first discuss the change you wish
+to make via an issue in the relevant repository for your change or email to the
+`keylime mailing list <https://groups.io/g/keylime>`_
```

Comment:
I didn't even know we had a mailing list...

Reply:
The mailing list is pretty inactive, but exists. Maybe point them to the Slack channel instead?

Reply:
I'll change the documentation to point to the Slack channel